api/v1/lib/resourcetest/resourcetest.go

```diff
@@ -72,7 +72,7 @@ func Reservation(ri *mesos.Resource_ReservationInfo) Opt {
 	}
 }
 
-func Disk(persistenceID, containerPath string) Opt {
+func Disk(persistenceID, containerPath, source string, sourceType mesos.Resource_DiskInfo_Source_Type) Opt {
```

Comment:
nit: would rather have left the params for `Disk` alone and created a new `DiskWithSource` func w/ the extended params. Callers of `Disk(x, y, "", 0)` shouldn't actually need to bother with the additional params.